corehq/apps/export/dbaccessors.py

```diff
@@ -214,3 +217,10 @@ def delete_all_export_instances():
     for row in db.view('export_instances_by_domain/view', reduce=False):
         doc_id = row['id']
         safe_delete(db, doc_id)
+
+
+def get_ledger_section_entry_combinations(domain):
+    """Get all section / entry combinations in a domain.
+    :returns: a generator of namedtuples with fields ``section_id``, ``entry_id``, ``doc_count``
+    """
+    return LedgerAccessors(domain).get_section_and_entry_combinations()
```

Comment:
Usually I would have just added this one line of code to where its used, but it gets mocked https://github.com/dimagi/commcare-hq/blob/ccc07728a9df852c22a0c1ab7aac92aa38e52e46/corehq/apps/export/tests/test_export_column.py#L191. Didn't seem worth it to change the mock to avoid the function